wallet/how-to/use-sdk/gaming/unreal-engine.md

```diff
@@ -6,5 +6,5 @@ sidebar_position: 2
 # Use MetaMask SDK with Unreal Engine
 
 [MetaMask SDK](../../../concepts/sdk/index.md) support for Unreal Engine games is coming soon.
```

Comment:
Should we link to the Unreal engine games page for the time being? https://www.unrealengine.com/en-US/solutions/games

Reply:
Same question for the Wagmi mention. Should that include a link as well to the official page? https://wagmi.sh